lib/content_item_loader.rb

```diff
@@ -38,6 +38,7 @@ def load_from_sources(base_path)
   end
 
   def load_from_graphql(base_path)
+    set_prometheus_labels("graphql_status_code" => 200, "graphql_contains_errors" => false, "graphql_api_timeout" => false)
```

Comment:
Call this here feels a bit like false reporting. I think this should be set after the call to publishing-api so it can accurately report the status codes and the lack of errors.

Reply:
So it doesn't actually send anything to prometheus until further down the line when the prometheus exporter is invoked. [Here it's just building up an object to send later](https://github.com/alphagov/frontend/pull/4851/files#diff-4c2ac1f535a1643c98d54d305323a278558c08f9309b72e18c224179575d02ddR104-R108). Setting it here and waiting for it to be overwritten by the calls to publishing API failing in some way seemed like the cleanest way to do it, but either way we won't be sending anything false over to prometheus

Reply:
Perhaps the logic can be re-thought?